gds_py/Dockerfile

```diff
@@ -40,7 +43,12 @@ RUN conda install -c conda-forge -c defaults --quiet --yes \
      'statsmodels' \
      'urbanaccess' \
      'xlrd' \
-     'xlsxwriter'
+     'xlsxwriter' \
+     && conda clean --all --yes --force-pkgs-dirs \
+     && find /opt/conda/ -follow -type f -name '*.a' -delete \
+     && find /opt/conda/ -follow -type f -name '*.pyc' -delete \
+     && find /opt/conda/ -follow -type f -name '*.js.map' -delete \
```

Comment:
This will ensure a smaller image by removing stuff that you'll probably won't use.